pkg/bundlemanager/util.go

```diff
@@ -0,0 +1,105 @@
+package bundlemanager
+
+import (
+	"sort"
+	"strings"
+)
+
+var bundlePrefix = "bundles.validator.forsetisecurity.org/"
+
+// BundleAnnotation
+func BundleAnnotation(key string) bool {
+	return strings.HasPrefix(key, bundlePrefix)
+}
+
+func bundleControls(obj Object) map[string]string {
+	tags := map[string]string{}
+	for k, v := range obj.GetAnnotations() {
+		if strings.HasPrefix(k, bundlePrefix) {
+			tags[k] = v
+		}
+	}
+	return tags
+}
+
+func allBundles(objs []Object) []string {
+	var tags []string
+	for _, obj := range objs {
+		for k := range bundleControls(obj) {
+			tags = append(tags, k)
+		}
+	}
+	return uniqueSorted(tags)
```

Comment:
Instead of implementing our own sorting mechanism, could we use a set library?

Reply:
Go doesn't have generics, so for stuff like this it's typically easier to do it manually over and over for specific types, or more poetically, live by the fast compilation, die by the fast compilation.

Reply:
Got it, this works for me then.